resources/language/German/strings.po

```diff
@@ -474,7 +474,7 @@ msgstr "Empfange Wiedergabefortschritt der Episoden von Trakt"
 
 msgctxt "#32120"
 msgid "Parsing episode playback progress %i of %i from Trakt"
-msgstr "Verarbeite Wiedergabefortschritt der Episoden %i of %i von Trakt"
+msgstr "Verarbeite Wiedergabefortschritt der Episoden %i von %i von Trakt"
```

Comment:
#### So either this: 
"Verarbeite Wiedergabefortschritt der Episoden %i von %i von Trakt"
#### Or this:
"Trakt verarbeitet Wiedergabefortschritt der Episoden %i von %i" 
?
***

Sorry for the pull request...I never intended to bother you, just didn't read first - fml I guess...